frontend/passkeys.js

```diff
@@ -58,6 +60,53 @@ export async function authenticatePasskey(targetHref, csrfToken) {
     }
 }
 
+export function deletePasskey(passkeyName, csrfToken) {
+    try {
+        if (confirm(`Are you sure you want to delete the passkey "${passkeyName}"?`)) {
+            createAndSubmitForm('/passkey/delete', {
+                passkeyName: passkeyName,
+                csrfToken: csrfToken
+            });
+        }
+    } catch (err) {
+        console.error('Error during passkey deletion:', err);
+        M.toast({ html: 'Failed to delete passkey. Please try again.', classes: 'rounded red' });
+    }
+}
+
+export function setUpDeletePasskeyButtons(selector) {
+    const deleteButtons = document.querySelectorAll(selector);
+    if (!deleteButtons.length) return;
+
+    deleteButtons.forEach(button => {
+        button.addEventListener('click', async () => {
+            const passkeyName = button.getAttribute('data-passkey-name');
```

Comment:
This looks redundant now

Reply:
We actually need this attribute in the delete method for the confirmation text. But I've removed the duplication that was also in the deletePasskey function.